meta-intel-edison-distro/recipes-core/images/edison-image.bb

```diff
@@ -75,6 +75,7 @@ IMAGE_INSTALL:append = " nodejs-npm"
 IMAGE_INSTALL:append = " libiio"
 IMAGE_INSTALL:append = " libiio-iiod"
 IMAGE_INSTALL:append = " libiio-python3"
+IMAGE_INSTALL:append = " libiio-tests"
```

Comment:
Commit message says about iiod, which is enabled already according to this diff.